### ISSUE TITLE
Atomic Swap - Display QRcode

### ISSUE THREAD

**Issue by vekamo** [mwcproject/mwc-qt-wallet#814]
It would be great for the sender of the coin like BTC, BCH ect to get a QR Code in the notification page when he's sending the coin to the lock address.

**Comment by bayk** [comment]
@vek-kal ,  what exactly do you want in QT code?  Just an address where to send the funds?
We can show qr code at the message box dialog.

**Comment by vek-kal** [comment]
Yes, that would exactly make it 👍 showing the lock address in string and next to it, a QR code button to show. If possible as well a copy button.
This gif shows a possible copy and QR Code button disposal in the box dialog, would be great to make something similar.

https://user-images.githubusercontent.com/80525364/111248904-bdf5ef00-8602-11eb-8d76-b99f21ad3e36.mp4

**Comment by bayk** [comment]
Yes, make sense. Currently user need to select the the locking address and  then copy it.
